packages/suite/src/views/wallet/anonymize/components/CoinjoinSetupStrategies.tsx

```diff
@@ -67,30 +69,49 @@ const StyledTooltipButton = styled(TooltipButton)`
 `;
 
 interface CoinjoinSetupStrategiesProps {
-    account: Account;
+    selectedAccount: SelectedAccountLoaded;
 }
 
-export const CoinjoinSetupStrategies = ({ account }: CoinjoinSetupStrategiesProps) => {
+export const CoinjoinSetupStrategies = ({ selectedAccount }: CoinjoinSetupStrategiesProps) => {
     const [strategy, setStrategy] = useState<CoinJoinStrategy>('recommended');
+    const [customMaxFee, setCustomMaxFee] = useState(3);
+    const [customSkipRounds, setCustomSkipRounds] = useState(true);
     const [connectedConfirmed, setConnectedConfirmed] = useState(false);
     const [termsConfirmed, setTermsConfirmed] = useState(false);
 
     const actions = useActions({
         startCoinjoinSession,
     });
+    const coordinatorData = useSelector(
+        state => state.wallet.coinjoin.clients[selectedAccount.network.symbol],
+    );
+    const targetAnonymity = useSelector(selectCurrentTargetAnonymity);
+    const { notAnonymized } = useSelector(selectCurrentCoinjoinBalanceBreakdown);
+
+    if (!coordinatorData || targetAnonymity == null) {
+        return null;
+    }
 
     const isCustom = strategy === 'custom';
     const allChecked = connectedConfirmed && termsConfirmed;
-    const fee = (Number(account.balance) * 0.003).toString();
+    const fee = new BigNumber(notAnonymized).times(coordinatorData.coordinatorFeeRate).toString();
 
     const reset = () => setStrategy('recommended');
     const toggleConnectConfirmation = () => setConnectedConfirmed(current => !current);
     const toggleTermsConfirmation = () => setTermsConfirmed(current => !current);
     const anonymize = () =>
-        actions.startCoinjoinSession(
-            account,
-            COINJOIN_STRATEGIES[strategy === 'custom' ? 'recommended' : strategy], // TODO: enable custom strategy
-        );
+        actions.startCoinjoinSession(selectedAccount.account, {
+            maxCoordinatorFeeRate:
+                coordinatorData.coordinatorFeeRate * 100 * 10 ** selectedAccount.network.decimals, // fee rate to percent to satoshis
```

Comment:
if you multiply by 100 in the `transformCoinjoinStatus` the you don't have to multiply it here and you can use "percentage format" anywhere else in components

it also would be nice to use some utilit we already have like `amountToSatoshi` but it returs a string :/

Reply:
Yes, that is why I did not use `amountToSatoshi`...